examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    outputs = user_id+item_id+targets+add_feat+te_feat
</pre><p>nit: Do you mind adding a space on either side of the <code>+</code> operator? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>